pkg/server/server.go

```diff
@@ -127,7 +127,6 @@ func (s *Server) Start() {
 		}
 
 		edv2 := router.PathPrefix("/api/v2/edgedevice").Subrouter()
-		edv2.Use(ensureMTLS)
```

Comment:
If we are getting rid of `ensureMTLS` usage, should we not delete the unused `func` as well?

Reply:
We use it for v1. V1 has no AuthContainer and rely on mTLS

Reply:
So v1 still is supported? That makes sense. Where does it then ensure that we have mTLS if using v1?

Reply:
Few lines to the up you can find `ed.Use(ensureMTLS)`